README.md

```diff
@@ -69,71 +68,42 @@ output "efs_backup_security_group" {
 
 
 
-## Integration with `EFS`
+## Integration with `mongo`
 
-To enable connectivity between the `DataPipeline` instances and the `EFS`, use one of the following methods to configure Security Groups:
+To enable connectivity between the `DataPipeline` instances and the `mongo`, use one of the following methods to configure Security Groups:
 
-1. Explicitly add the `DataPipeline` SG (the output of this module `security_group_id`) to the list of the `ingress` rules of the `EFS` SG. For example:
+1. Explicitly add the `DataPipeline` SG (the output of this module `security_group_id`) to the list of the `ingress` rules of the `mongo` SG. For example:
 
 ```hcl
-module "elastic_beanstalk_environment" {
-  source     = "git::https://github.com/cloudposse/terraform-aws-elastic-beanstalk-environment.git?ref=master"
-  namespace  = "${var.namespace}"
-  name       = "${var.name}"
-  stage      = "${var.stage}"
-  delimiter  = "${var.delimiter}"
-  attributes = ["${compact(concat(var.attributes, list("eb-env")))}"]
-  tags       = "${var.tags}"
-
-  # ..............................
-}
-
-module "efs" {
-  source     = "git::https://github.com/cloudposse/terraform-aws-efs.git?ref=tmaster"
-  namespace  = "${var.namespace}"
-  name       = "${var.name}"
-  stage      = "${var.stage}"
-  delimiter  = "${var.delimiter}"
-  attributes = ["${compact(concat(var.attributes, list("efs")))}"]
-  tags       = "${var.tags}"
-
-  # Allow EB/EC2 instances and DataPipeline instances to connect to the EFS
-  security_groups = ["${module.elastic_beanstalk_environment.security_group_id}", "${module.efs_backup.security_group_id}"]
-}
 
-module "efs_backup" {
-  source     = "git::https://github.com/cloudposse/terraform-aws-efs-backup.git?ref=master"
+module "mongo_backup" {
+  source     = "git::https://github.com/ITSvitCo/terraform-aws-mongo-backup.git?ref=master"
   name       = "${var.name}"
-  stage      = "${var.stage}"
+  stage      = "${terraform.workspace}"
   namespace  = "${var.namespace}"
   delimiter  = "${var.delimiter}"
-  attributes = ["${compact(concat(var.attributes, list("efs-backup")))}"]
+  attributes = ["${compact(concat(var.attributes, list("mongo-backup")))}"]
```

Comment:
should be `attributes = ["${compact(concat(var.attributes, list("mongo"), list("backup")))}"]`